### ISSUE TITLE
"Rule not found" reported as "Rule is not a constructor"

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#982]
I've forgotten to `yarn install` after CI updated the dep to v2.0.0-beta.3 and was still on v2.0.0-beta.2. When I tried to enable the `require-button-type` rule and ran `yarn lint:hbs` I was seeing error messages like this:

```
/Users/tbieniek/Code/foo/app/bar/template.hbs
  -:-  error  Rule is not a constructor  undefined
```

I'm not sure if this is a regression in v2.0.0 or if `master` has the same behavior, but we should display a better error message if a rule does not actually exist 😉

**Comment by rwjblue** [comment]
Ya, totally agreed.

**Comment by amk221** [comment]
`require-button-type` is my new favourite rule.

**Comment by rwjblue** [comment]
He was using a version of ember-template-lint that didn't have it yet, this issue is reporting that our error messages for missing rules (e.g. when you specify a config manually for a rule that we can't find) is horrible.

**Comment by amk221** [comment]
Right yes, my bad. I had the wrong version.
